docs/src/modules/java/pages/call-another-as-service.adoc

```diff
@@ -0,0 +1,118 @@
+= Calling other Akka Serverless services
+
+// FIXME this does not actually work (yet?) https://github.com/lightbend/akkaserverless-docs/issues/903#issuecomment-919174111
+
+// FIXME possible from the same project only?
+In some cases it is useful to call a component in another service, for example to write logic that touches multiple entities or actions, to aggregate their return values or perform a multi-step workflow.
+
+Calling other Akka Serverless services from an Action is done by invoking them over gRPC much like how an external client would. The service is however identified by its service name and encrypting the communication using TLS is handled for us by Akka Serverless.
+
+It is possible to use any gRPC client you want, but since Akka gRPC is already used by the SDK we recommend using it.
+
+Since the proto file of an Akka Serverless service contains annotations that cause the Java SDK code generation to generate services we need to start by removing those. Copy the API definition proto file from the other service into the `proto` directory but strip all `akkaserverless.service` option blocks as well as all other Akka Serverless annotations and the `import "akkaserverless/annotations.proto"` from it.
+
+The build needs to be updated to generate Akka gRPC client classes. In the existing configuration for the `protobuf-maven-plugin` in your `pom.xml` add the following:
+
+---
+// FIXME this block is a bit unwieldy, but har to understand where/what to add if not
+// FIXME should we _always_ include something instead, like a src/main/ex-proto and avoid clients for the service itself?
```

Comment:
I think we need clients for integration tests. I don't know why we have to be restrictive in generating clients. Doesn't hurt to have all of them?

Now we have `grpc-client` execution for `test-compile` in the pom.xml. We could have that for `compile` by default?

Reply:
Yeah, sure, we could, I guess it is about putting stuff in the prod classpath that will actually never be used, but might not matter _that_ much.

Reply:
All samples and archetypes now updated to generate Akka gRPC clients